controls/V-92975.rb

```diff
@@ -92,7 +92,7 @@
           date_difference = expiration_date.mjd - creation_date.mjd
           describe "Account expiration set for #{account_name}" do
             subject { date_difference }
-            it { should cmp <= input('temporary_account_period') }
+            it { should cmp <= input('temporary_account_period')/24 }
```

Comment:
Should we wrap this in a `floor` - https://www.geeksforgeeks.org/ruby-integer-floor-function-with-example/

Reply:
I would prefer to leave the test itself alone since inaccuracies could potentially occur if we muck with the number before the comparison (see my most recent pr against the compliance chart).  If we want to improve the clarity of the results, then I'd rather use the other syntax for writing controls so as to provide an explicit failure message.

Reply:
Some ARS controls would require 12 hours (0.5 days) as the value - in cases like that, using floor/ceiling would lead to an incorrect test (please let me know if I'm wrong) @aaronlippold

Reply:
Aaron agreed to keep it like it is on Slack